Source/WebCore/rendering/style/RenderStyle.h

```diff
@@ -2481,6 +2485,9 @@ class RenderStyle final : public CanMakeCheckedPtr<RenderStyle> {
 
     DataRef<SVGRenderStyle> m_svgStyle;
 
+    // list of associated pseudo styles
+    std::unique_ptr<RenderLayerFilters> m_layerFilters;
```

Comment:
Given how uncommon filter usage is, I am surprised to see this on base RenderStyle object itself. I also don't understand the comment.